code/game/area/general.dm

```diff
@@ -10,6 +10,12 @@
 	temperature = TCMB
 	pressure = 0
 
+/area/space/Initialize(mapload)
+	. = ..()
+	if(mapload)
+		var/area/ourarea = get_area(src)
+		ENABLE_BITFIELD(ourarea.flags_area, NO_DROPPOD)
```

Comment:
no need. 

you can do the same but better by changing the var on compilation

/area/space
	name = "Space"
	requires_power = 1
	always_unpowered = 1
	dynamic_lighting = DYNAMIC_LIGHTING_ENABLED
	power_light = 0
	power_equip = 0
	power_environ = 0
	ambience = list('sound/ambience/ambispace.ogg')
	temperature = TCMB
	pressure = 0
    flags_area = NO_DROPPOD <------

Reply:
Use codeblocks, but yea basically this